pkg/capabilities/consensus/ocr3/capability_test.go

```diff
@@ -165,11 +163,22 @@ func TestOCR3Capability_Eviction(t *testing.T) {
 		Inputs: inputs,
 	}
 
-	callback, err := cp.Execute(ctx, executeReq)
-	require.NoError(t, err)
-
-	fc.Advance(1 * time.Hour)
-	resp := <-callback
+	go func() {
```

Comment:
Let's track this and block so it doesn't leak. Something like this would work:
```suggestion
	done := make(chan struct{})
	t.Cleanup(func() { <-done })
	go func() {
		defer close(done)
```

Reply:
looks like the test context is cancelled after t.Cleanup is called, so creating a child context from the test context and explicitly cancelling that in the test.